src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -1931,7 +1931,7 @@ static void copy(MemorySegment srcSegment, ValueLayout srcElementLayout, long sr
      * layout size.
      *
      * @param layout the layout of the region of memory to be read
-     * @param index a logical index, the offset in bytes (relative to this
+     * @param index a logical index denoting the offset in bytes (relative to this
```

Comment:
apologies for sending you the wrong way. As a single sentence this doesn't make sense. I suggest the original text (e.g. before this PR) is restored.